manifest.toml

```diff
@@ -53,6 +53,15 @@ ram.runtime = "50M"
 
     [resources.ports]
 
+    [resources.sources]
+    
+        [resources.sources.main]
+        url = "https://github.com/n8n-io/n8n/archive/refs/tags/n8n@1.9.3.zip"
+        sha256 = "53114a7817f319bfc065566cd77ae889aacb2f2ffd8283f905292a31e19d41e1"
+        autoupdate.strategy = "latest_github_release"
+        autoupdate.version_regex = "^n8n@(.*)$"
+        prefetch = false
```

Comment:
(Not sure why disable prefetch but okay :grimacing: )

Reply:
It's because we actually install n8n with `$ynh_npm install n8n@${n8n_version}` where `n8n_version=$(ynh_app_upstream_version)`, so we don't actually need to download the package.